exercises/Graphing-3-R.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: exercise
+title: Graphing 3
+language: R
+---
+
+It makes sense that larger organisms have larger offspring, but what the
+mathematical form of this relationship should be is unclear. Let's
+look at the problem empirically for mammals.
+
+Download some
+[mammal life history data](http://esapubs.org/archive/ecol/E084/093/Mammal_lifehistories_v2.txt) from the web. 
+You can do this either directly in the program using `read.csv()` or download 
+the file to your computer using your browser and import it from there.
+
+When you import the data there are some extra blank lines at
+the end of this file. Get rid of them by using the optional `read.csv()`
+argument to select the valid 1440 rows, `nrows = 1440`.
+
+Missing data in this file is specified by -999 and -999.00. Tell R that
+these are null values using the optional `read_csv()` argument,
```

Comment:
>> read.csv() vs read_csv() should be consistent.
Done. `read_csv()` -> `read.csv()`. `read_csv()` is just wrong.

On Thu, Jun 11, 2015 at 3:26 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Graphing-3-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/57#discussion_r32264233>
> :
>
> > +
> > +It makes sense that larger organisms have larger offspring, but what the
> > +mathematical form of this relationship should be is unclear. Let's
> > +look at the problem empirically for mammals.
> > +
> > +Download some
> > +[mammal life history data](http://esapubs.org/archive/ecol/E084/093/Mammal_lifehistories_v2.txt) from the web.
> > +You can do this either directly in the program using `read.csv()` or download
> > +the file to your computer using your browser and import it from there.
> > +
> > +When you import the data there are some extra blank lines at
> > +the end of this file. Get rid of them by using the optional `read.csv()`
> > +argument to select the valid 1440 rows, `nrows = 1440`.
> > +
> > +Missing data in this file is specified by -999 and -999.00. Tell R that
> > +these are null values using the optional `read_csv()` argument,
>
> read.csv() vs reac_csv() should be consistent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/57/files#r32264233>
> .
>